update.sh

```diff
@@ -13,9 +13,9 @@ defaultDebianSuite='buster-slim'
 declare -A debianSuite=(
 	#[1.6]='stretch-slim'
 )
-defaultAlpineVersion='3.12'
+defaultAlpineVersion='3.13'
 declare -A alpineVersion=(
-	#[1.6]='3.8'
+	[1.7]='3.12'
```

Comment:
This is interesting - was there a build failure on HAProxy 1.7?
(maybe a one-liner error we could include in a comment here?)

Reply:
```
multiple definition of `pool2_trash'; src/haproxy.o:(.bss+0x0): first defined here
collect2: error: ld returned 1 exit status
```
Alpine 3.13 upgraded to GCC 10